mteb/tasks/Classification/mya/MyanmarNews.py

```diff
@@ -0,0 +1,51 @@
+from __future__ import annotations
+
+from mteb.abstasks import AbsTaskClassification
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+
+class MyanmarNews(AbsTaskClassification):
```

Comment:
I think this would probably be better formulated as thematic clustering instead of classification. That way we can also avoid the hustle of training a classifier.

Reply:
Ohh okay. Shall move it to the Clustering directory and remove it from Classifcation?

Reply:
I have used AbsTaskClustering, but I'm getting this error and am not sure what exactly is wrong - 
File "C:\Users\HP\OneDrive\Desktop\OpenSource\mteb\mteb\tasks\Clustering\mya\BurmeseClustering.py", line 7, in <module>
    class BurmeseClustering(AbsTaskClusteringFast):
TypeError: module() takes at most 2 arguments (3 given)

Reply:
it's because you need to import it like this:
```python
from abstasks.AbsTaskClusteringFast import AbsTaskClusteringFast
```
instead of:
```python
from abstasks import AbsTaskClusteringFast
```

(((don't ask why)))

Reply:
I've tried to change the code to match the format for Clustering but I've been getting this error -  
  File "C:\Users\HP\OneDrive\Desktop\OpenSource\mteb\venv\Lib\site-packages\datasets\formatting\formatting.py", line 521, in _check_valid_column_key
    raise KeyError(f"Column {key} not in the dataset. Current columns in the dataset: {columns}")
KeyError: "Column labels not in the dataset. Current columns in the dataset: ['sentences', 'label']"

To be frank, I was focusing on adding classification datasets and formatted all my code to that, so I'm a bit lost about how to make it work for Clustering